### PR TITLE
fix pg_stat_monitor

### PR DESCRIPTION
**Pull request by hors**
* fix PMM_AGENT_PRERUN_SCRIPT

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-316/846e6692/init-deploy.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-316/846e6692/demand-backup.log) |
| start-from-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-316/846e6692/start-from-backup.log) |
| monitoring | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-316/846e6692/monitoring.log) |
| We run 4 out of 4|

commit: https://github.com/percona/percona-postgresql-operator/pull/316/commits/846e6692ccf448750942ab249dea796e788edb31
image: `perconalab/percona-postgresql-operator:PR-316-846e6692`

File: internal/postgres/hba.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ func NewHBAs() HBAs {
 	return HBAs{
 		Mandatory: []HostBasedAuthentication{
 			// The "postgres" superuser must always be able to connect locally.
-			*NewHBA().Local().User("postgres").Method("peer"),
+			*NewHBA().Local().User("postgres").Method("trust"),
```
Comment (egegunes) [inline internal/postgres/hba.go:28]: why this change?

**Review by egegunes — CHANGES_REQUESTED** [review]

File: internal/postgres/hba.go (hunk 1 above)
Comment (hors) [reply internal/postgres/hba.go:28]: because we can't connect via socket from pmm-client. Anyway I think it is not good a approach to change it and we need to improve it in the next release
```
peer
Obtain the client's operating system user name from the operating system and check if it matches the requested database user name. This is only available for local connections. See [Section 21.9](https://www.postgresql.org/docs/current/auth-peer.html) for details.
```

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| We run 0 out of 4|

commit: https://github.com/percona/percona-postgresql-operator/pull/316/commits/9128ccfa3b85d08fb7b94290b39337a2bd2498b3
image: `perconalab/percona-postgresql-operator:PR-316-9128ccfa`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-316/9128ccfa/init-deploy.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-316/9128ccfa/demand-backup.log) |
| start-from-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-316/9128ccfa/start-from-backup.log) |
| monitoring | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-316/9128ccfa/monitoring.log) |
| We run 4 out of 4|

commit: https://github.com/percona/percona-postgresql-operator/pull/316/commits/9128ccfa3b85d08fb7b94290b39337a2bd2498b3
image: `perconalab/percona-postgresql-operator:PR-316-9128ccfa`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-316/6255029f/init-deploy.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-316/6255029f/demand-backup.log) |
| start-from-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-316/6255029f/start-from-backup.log) |
| monitoring | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-316/6255029f/monitoring.log) |
| We run 4 out of 4|

commit: https://github.com/percona/percona-postgresql-operator/pull/316/commits/6255029fa463ad658a950e39b1e53f391038852c
image: `perconalab/percona-postgresql-operator:PR-316-6255029f`

**Review by egegunes — APPROVED** [review]
